### PR TITLE
Adds HP regen to Score and Stat commands

### PR DESCRIPTION
**Pull request by daedela**

### CODE REVIEW

File: src/act.wizinfo.cpp [hunk 1]
```diff
@@ -667,11 +667,10 @@ void do_stat_character(CharData *ch, CharData *k) {
...
         CLR(ch, ANRM));
 
-    resp += fmt::format("HP: [{}{}/{}+{}{}]  MP: [{}{}/{}+{}{}]  MV: [{}{}/{}+{}{}]\n", CLR(ch, FGRN), GET_HIT(k),
-                        GET_MAX_HIT(k), hit_gain(k), CLR(ch, ANRM), CLR(ch, FGRN), GET_MANA(k), GET_MAX_MANA(k),
-                        mana_gain(k), CLR(ch, ANRM), CLR(ch, FGRN), GET_MOVE(k), GET_MAX_MOVE(k), move_gain(k),
-                        CLR(ch, ANRM));
-
+    resp += fmt::format("HP: [{}{}/{}{}]  HP Gain: [{}{}{}] HP Regen Bonus: [{}{}{}]\n", 
+        CLR(ch, FGRN), GET_HIT(k), GET_MAX_HIT(k), CLR(ch, ANRM), CLR(ch, FGRN), hit_gain(k), CLR(ch, ANRM), CLR(ch, FGRN), k->char_specials.hitgain, CLR(ch, ANRM));
+    resp += fmt::format("MV: [{}{}/{}{}]  MP Gain: [{}{}{}]\n", CLR(ch, FGRN), GET_MOVE(k), GET_MAX_MOVE(k), CLR(ch, ANRM), CLR(ch, FGRN), move_gain(k), CLR(ch, ANRM));
```
Comment (stridera) [inline src/act.wizinfo.cpp:672]: Should be MV gain

File: src/act.wizinfo.cpp [hunk 2]
```diff
@@ -667,11 +667,10 @@ void do_stat_character(CharData *ch, CharData *k) {
...
+    resp += fmt::format("MP: [{}{}/{}{}]  MP Gain: [{}{}{}]\n", CLR(ch, FGRN), GET_MANA(k), GET_MAX_MANA(k), CLR(ch, ANRM), CLR(ch, FGRN), mana_gain(k), CLR(ch, ANRM));
```
Comment (stridera) [inline src/act.wizinfo.cpp:673]: I'd just remove this line.  I'm going to completely strip mana after we finish the spell slot update.

File: src/act.wizinfo.cpp (hunk 1 above)
Comment (daedela) [reply src/act.wizinfo.cpp]: Fixed

File: src/act.wizinfo.cpp (hunk 2 above)
Comment (daedela) [reply src/act.wizinfo.cpp]: Removed

**Review by stridera — APPROVED** [review]

**Pull request merged**